src/widgets/PrescriptionInfo.js

```diff
@@ -58,7 +65,11 @@ const PrescriptionInfoComponent = ({
 
           <FlexRow flex={1}>
             <CircleButton IconComponent={HistoryIcon} onPress={viewHistoryCallback} />
-            <CircleButton IconComponent={PencilIcon} onPress={editPatientCallback} />
+            <CircleButton
+              IconComponent={PencilIcon}
+              isDisabled={!canEditPatient}
+              onPress={editPatientCallback}
+            />
```

Comment:
I think this is the wrong way to go about this. I think you should still be able to press this button to bring up the patient details, but the form should be in read only. I thought it was already doing this?

That way you can always have access and see all of the patients details